lib/github_service/commands/run_test.rb

```diff
@@ -0,0 +1,278 @@
+module GithubService
+  module Commands
+    # = GithubService::Commands::RunTest
+    #
+    # Triggers a build given the configured test repo:
+    #
+    #   Settings.run_tests_repo.*
+    #
+    # Which doing so will:
+    #
+    #    - validate the command
+    #    - create a branch
+    #    - create a commit with the travis.yml changes
+    #    - push said branch to the origin
+    #    - create a pull request for that branch
+    #
+    # More info can be found here on how the whole process works:
+    #
+    #   https://github.com/ManageIQ/manageiq-cross_repo-tests
+    #
+    # == Command structure
+    #
+    # @miq-bot run-tests [<repos-to-test>] [including <extra-repos>]
+    #
+    # where:
+    #
+    #   - a `repo` is of the form [org/]repo[@ref|#pr]
+    #   - `repos-to-test` is a list of repos to have tested
+    #   - `extra-repos` is a list of repos (gems) to override in the bundle
+    #
+    # each "lists of repos" should be comma delimited.
+    #
+    # == Example
+    #
+    # In ManageIQ/manageiq PR #1234
+    #
+    #   @miq-bot run-tests manageiq-api,manageiq-ui-classic#5678 \
+    #     including Fryguy/more_core_extensions@feature,Fryguy/linux_admin@feature
+    #
+    # will create a commit with the .travis.yml changes:
+    #
+    #   @@ -14,6 +14,6 @@ matrix:
+    #      fast_finish: true
+    #    env:
+    #      global:
+    #   -  - REPOS=
+    #   +  - REPOS=Fryguy/more_core_extensions@feature,Fryguy/linux_admin@feature,ManageIQ/manageiq#1234
```

Comment:
This should also probably have `manageiq-ui-classic#5678`, otherwise it's not an accurate test?

Reply:
Oh yeah, yup.

I think this was a modified form of what you had in https://github.com/ManageIQ/miq_bot/issues/446 but I flubbed combining them together in a single example.  Will fix.

Reply:
Fixed.